ecs-cli/modules/cli/compose/project/project_parseV3_test.go

```diff
@@ -94,11 +140,123 @@ services:
 	project.ecsContext.ComposeFiles = append(project.ecsContext.ComposeFiles, tmpfile1.Name())
 	project.ecsContext.ComposeFiles = append(project.ecsContext.ComposeFiles, tmpfile2.Name())
 
-	// assert number of container configs matches expected number of services
-	// TODO: assert containerConfig content after conversion implemented
+	// get map of docker-parsed ServiceConfigs
+	expectedConfigs := getServiceConfigMap(t, project.ecsContext.ComposeFiles)
+
+	// assert # and content of container configs matches expected services
 	conConfigResult, err := project.parseV3()
 	if err != nil {
 		t.Fatal("Unexpected error parsing file: ", err)
 	}
 	assert.Equal(t, numOfServices, len(*conConfigResult))
+	for _, conConfig := range *conConfigResult {
+		verifyConvertToContainerConfigOutput(t, expectedConfigs[conConfig.Name], conConfig)
+	}
+}
+
+func verifyConvertToContainerConfigOutput(t *testing.T, observed types.ServiceConfig, expected containerconfig.ContainerConfig) {
```

Comment:
The naming of the parameters here is a tad confusing -- when `verifyConvertToContainerConfigOutput` is called, it seems to take `expected` first and then the observed/actual config second. Here, the naming appears to be reversed, which means the assertions are also reversed?

Reply:
ugh, yes these are confusing - originally i had them swapped but switched them when I realized error output reads "Expected [] But Saw []". Seems I forgot to swap the func params - i'll try to make the naming more explicit as well